ReactAndroid/src/main/java/com/facebook/react/uimanager/TouchTargetHelper.java

```diff
@@ -17,9 +17,13 @@
 import android.view.View;
 import android.view.ViewGroup;
 
+import com.facebook.react.R;
```

Comment:
FYI, this broke buck build.
Could you add the used resource to BUCK file in this folder?